Example/Tests/GiniVisionDocumentTests.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  GINIFilePickerManagerTests.swift
+//  GiniVision_Tests
+//
+//  Created by Enrique del Pozo Gómez on 8/30/17.
+//  Copyright © 2017 CocoaPods. All rights reserved.
+//
+
+import XCTest
+@testable import GiniVision
+
+
+class GINIVisionDocumentTests: XCTestCase {
+    
+    var filePickerManager:FilePickerManager {
+        return FilePickerManager()
+    }
+    
+    func testExcedeedMaxFileSize() {
+        // Given
+        let higherThan10MBData = generateFakeData(megaBytes: 12)
+        
+        // When
+        let fakeDocument = GiniPDFDocument(data: higherThan10MBData)
+        
+        // Then
+        XCTAssertThrowsError(try fakeDocument.validate(), "should indicate that is not a valid document") { error in
+            XCTAssert(error as? DocumentValidationError == DocumentValidationError.exceededMaxFileSize, "should indicate that max file size has been exceeded")
+        }
+    }
+    
+    func testNotExcedeedMaxFileSize() {
+        // Given
+        let lowerThanOrEqualTo10MBData = generateFakeData(megaBytes: 10)
+        
+        // When
+        let fakeDocument = GiniPDFDocument(data: lowerThanOrEqualTo10MBData)
+
+        // Then
```

Comment:
Given... when... then... I really don't think it makes it more readable. It's quite obvious already. Unless you have a strong opinion against it, I'd remove it.

Reply:
I use to write tests adding those comments, because for me it is more readable and it helps me to split in different steps. But there is no problem if you want me to remove them.